### ISSUE TITLE
BROWSERSTACK_APP_ID returns custom_id if set

### ISSUE THREAD

**Issue by ustav** [browserstack/browserstack-fastlane-plugin#19]
TL;DR;

in `upload_to_browserstack_app_automate_action.rb`  BROWSERSTACK_APP_ID holds `app_url` or `custom_id` if it is set

BROWSERSTACK_APP_ID should always return `app_url` or at least there should be a variable that returns it.

Longer version.

in `upload_to_browserstack_app_automate_action.rb` BROWSERSTACK_APP_ID returns `app_url` which could be used to set the specific app for testing. `custom_id` could be used as an additional piece of information, but could not be used instead of `app_url`. 

Even more, `custom_id` is provided by the user so it is known from the beginning. `app_url` is generated on the server and could not be known in advance. There's no reason to return `custom_id` instead of `app_url`

Please change this behavior in `browserstack_helper.rb` in lines 48-52. 

`app_url` should always be returned as it is needed to run subsequent tests
